### PR TITLE
coafile.rst: Fix bad code markup

### PR DESCRIPTION
**Pull request by Naveenaidu**
The changes were made to the documentation of coafile.rst . 
There was a bad codeblock markup at "[section1] # setting1 is inherited setting2 = 2". 
Changed it to "[section1] # setting1 is inherited setting2 = 2::"

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by Naveenaidu** [comment]
Can someone please take a look at this. This is my first PR. And i hope to learn a lot. :)

**Comment by Naveenaidu** [comment]
@Vedarth Made the changes as mentioned. Can you please review it again. I apologize for the bad commit message.

File: Help/Getting_In_Touch.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ following things:
    and we will check it out. In case of long discussion, you should create
    a new issue with clearly defined task!
 -  Give us feedback. If you think we're doing something useless or something
-   amazing, let us know by dropping a message on gitter or a mail on our
+   amazing, let us weffwef by dropping a message on gitter or a mail on our
```
Comment (Vedarth) [inline Help/Getting_In_Touch.rst]: @Naveenaidu This is not related to the issue.

**Comment by Vedarth** [comment]
@Naveenaidu Read api.coala.io/en/latest/Developers/Git_Basics.html thoroughly. It is really very helpful. If you get some specific doubts feel free to ask them in gitter channel.

**Comment by Naveenaidu** [comment]
@Vedarth Squashed the commits as suggested. Can you please review the issue once again. Sorry it took some time. Was figuring out how squash works. :)

**Comment by Vedarth** [comment]
@Naveenaidu No problem. Thanks for your contribution.

**Comment by Vedarth** [comment]
Just use `git commit --amend`

**Comment by Naveenaidu** [comment]
@Vedarth It's not at all an inconvinience. Made the changes as requested. Please check.

File: Users/coafile.rst [hunk 2]
```diff
@@ -93,7 +93,8 @@ This is the same file without section inheritance::
 All settings must be part of a section, so don't do this for implicit
 inheritance (this is also deprecated behavior). Implicit inheritance
 was leading to a section automatically getting inherited to all other
-sections without semantically making sense.
+sections without semantically making sense::
+
```
Comment (Makman2) [inline Users/coafile.rst]: Why this blank line?

**Comment by Makman2** [comment]
Commit message:
1. Fixes -> Fix
2. No colons inside the issue reference (see other commits)

**Comment by Makman2** [comment]
unack e29a4bf0e9d2a1ba5ce0a98170182c186627db9a

**Comment by Naveenaidu** [comment]
@Makman2  Made changes as requested. Please check

**Comment by Makman2** [comment]
ack a74b8573168108321275e08699ee67f9bab65413

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/documentation/pull/532/commits/fcef94d3e63467a5fd897c2b80212b58f9069f34

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff
